build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 plugins {
-  id("uk.gov.justice.hmpps.gradle-spring-boot") version "5.3.0"
+  id("uk.gov.justice.hmpps.gradle-spring-boot") version "5.4.0"
   kotlin("plugin.spring") version "1.9.0"
```

Comment:
`5.4.0` bundles with kotlin 1.9.10 so this should really be upgraded at the same time.

Reply:
also `springdoc-openapi-starter-webmvc-ui` should be upgraded to `2.2.0` for the latest spring boot version

Reply:
@petergphillips sorted